src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -11,16 +11,19 @@
     <PlatformTarget>AnyCPU</PlatformTarget>
     <PackageTargetFramework>dotnet5.5</PackageTargetFramework>
     <UsePackageTargetRuntimeDefaults>true</UsePackageTargetRuntimeDefaults>
+    <ProjectGuid>{D4550556-4745-457F-BA8F-3EBF3836D6B4}</ProjectGuid>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(TargetsWindows)' != 'true' ">
     <DefineConstants>$(DefineConstants);MANAGED_SNI</DefineConstants>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(TargetsWindows)' == 'true' ">
```

Comment:
i have locate non windows file to be compile only for non windows. e.g (LocaleMapperApiHelper.cs). This only used for unix. so it will not be compile against windows.

this WINDOWS_OS constant need it to switch between windows / unix implementation.

I believe this WINDOWS_OS Constant will be used a lot as we will have couple separate implementation for WINDOWS and NON Windows and this #if condition will never be switch off as well.